### PR TITLE
Revise About page

### PR DESCRIPTION
**Pull request by nurupo**
All this talk about The Tox Project 501(c)(3) organization and bylaws is
misleading, given no one is working on registering The Tox Project as an
501(c)(3) organization, nor anyone is working on bylaws.

### CODE REVIEW

**Comment by robinlinden** [comment]
@zero-one has been working on by-laws.

**Comment by nurupo** [comment]
He was the only person working on it, but @zero-one has quit Tox in the beginning of April and no one has worked on them ever since, nor anyone seems to want to work on them.

**Comment by nurupo** [comment]
Github doesn't have this revision thing Reviewable has, huh.

**Comment by nurupo** [comment]
@JFreegman says that it's useful to have the table listing who runs the infrastructure. I guess we could leave the table in there until we do the Team page, which would list major Tox contributors: client developers, toxcore developers, package managers, website maintainers, etc. Similar to https://gentoo.org/inside-gentoo/developers/ or https://geti2p.net/en/about/team.

**Comment by nurupo** [comment]
Included the list. Sorted alphabetically. Also included @optimumtact on there. Screenshot: http://i.imgur.com/fSM5cjD.png

The list looks a bit weird on mobile though http://i.imgur.com/JJtDSkI.png -- the text is centered by the dots are on the left.

**Comment by robinlinden** [comment]
Add text-align: left; to the style of the table to fix the look on mobile.

**Comment by nurupo** [comment]
Given everything is centered on mobile, the last screenshot seems to look the best.

**Comment by anoadragon453** [comment]
@robinlinden Can you center the names and remove the black dots?

**Comment by robinlinden** [comment]
@anoadragon453 of course. :)

https://i.imgur.com/g2Xj2rM.png

Style for the `<ul>`:
```
max-width: 300px;
margin: 0 auto;
list-style: none;
text-align: center;
```

**Comment by anoadragon453** [comment]
Looks much better to me. Only thing I notice now is that it's off-center with the 'Learn More' title. Is that off center or is it the list of names?

**Comment by robinlinden** [comment]
Wasn't thinking about the fact that `ul` has a ton of left-padding by default. Good catch. 

[Fixed it.](https://i.imgur.com/tiiCB2C.png)

Edited the code in the previous comment to include the fix.

**Comment by anoadragon453** [comment]
Looks much nicer to me. @nurupo ?

**Comment by nurupo** [comment]
What are you guys even doing? Why are you removing dots in a list? You could just put the names each on a new line and get the same effect as removing dots in a list.

**Comment by nurupo** [comment]
I kind of want it to stay the list, that is, with the dots.

**Comment by anoadragon453** [comment]
Ill let @nurupo have the last say - he's done much more for this project than I have.

**Comment by nurupo** [comment]
@robinlinden could you provide CSS for centering the list similar while having the text still being left-aligned? Very similar to https://imgur.com/8BDzURw, except that in that screenshot the list is a bit off center, it's more to the left. You could open a PR in my fork, if you want.

**Comment by Zer0-One** [comment]
:eggplant:

File: themes/website/templates/about.html [hunk 1]
```diff
@@ -5,141 +5,61 @@
...
-            library implementation is nearing completion (in terms of
-            features). Tox (both core and clients) has thousands of users,
-            hundreds of contributors, and the project shows no sign of slowing
-            down. Recently, a group of some of the project's major contributors
-            have formed <b>The Tox Project</b>, an organization built around
-            the protection, promotion, and advancement of Tox and its development.</p>
+			<p class="lead">
+			Tox began a few years ago, in the wake of Edward Snowden's leaks regarding NSA
+			spying activity. The idea was to create an instant messaging application that
+			ran without any kind of central servers. The system would be distributed,
```
Comment (optimumtact) [inline themes/website/templates/about.html]: without requiring the use of

File: themes/website/templates/about.html [hunk 2]
```diff
@@ -5,141 +5,61 @@
...
+			peer-to-peer, and end-to-end encrypted, with no way to disable any of the
+			encryption features; at the same time, the application would be easily usable
+			by the layperson with no practical knowledge of cryptography or distributed
+			systems. During the Summer of 2013 a small group of developers from all around
+			the globe was formed and began working on a library implementing the Tox
```
Comment (optimumtact) [inline themes/website/templates/about.html]: formed

instead of 

    was formed

File: themes/website/templates/about.html [hunk 3]
```diff
@@ -5,141 +5,61 @@
...
+			protocol. The library provides all of the messaging and encryption facilities,
+			and is completely decoupled from any user-interface; for an end-user to make
+			use of Tox, they need a Tox client. Fast-forward a few years to today, and
+			there exist several independent Tox client projects, and the original Tox core
+			library implementation is continues improving. Tox (both core library and
```
Comment (optimumtact) [inline themes/website/templates/about.html]: continues to improve

File: themes/website/templates/about.html [hunk 4]
```diff
@@ -5,141 +5,61 @@
...
+			clients) has thousands of users, hundreds of contributors, and the project
+			shows no sign of slowing down.
+			</p>
+			<p class="lead">
+			Tox is a FOSS (Free and Open Source) project. All Tox code is open source and
+			all development occurs in the open. Tox is developed by volunteer developers who
+			spend their after work hours on it, believing in the idea of the project. Tox is
```
Comment (optimumtact) [inline themes/website/templates/about.html]: after work hours

change to

    free time

File: themes/website/templates/about.html [hunk 5]
```diff
@@ -5,141 +5,61 @@
...
+			<h2 class="header">Contact</h2>
+			<p class="lead">
+			This website and other infrastructure such as wiki, blog, etc. are maintained
+			by some of the long-time contributors of Tox. If you have a legal question, or
+			if you notice something is wrong with tox.chat servers, or you find an outdated
+			information on the one of tox.chat websites -- you can send an email to our
+			distribution list <a href="mailto:leadership@tox.chat">leadership@tox.chat</a>.
+			Please note that this email is not the right place to ask for general user
+			support. Please use our <a href="https://lists.tox.chat/listinfo">mailing lists
+			</a> or <a href="https://www.reddit.com/r/projecttox/">Reddit forums</a> for
```
Comment (optimumtact) [inline themes/website/templates/about.html]: or

should probably be

    or the

**Pull request merged**